TramsDataApi/DatabaseModels/AcademyConversionProject.cs

```diff
@@ -1,109 +1,115 @@
+      public string LocalAuthorityInformationTemplateLink { get; set; }
+      public bool? LocalAuthorityInformationTemplateSectionComplete { get; set; }
+
+      //school/trust info
+      public string RecommendationForProject { get; set; }
+      public string Author { get; set; }
+      public string Version { get; set; }
+      public string ClearedBy { get; set; }
+      public string AcademyOrderRequired { get; set; }
+      public DateTime? PreviousHeadTeacherBoardDate { get; set; }
+      public string PreviousHeadTeacherBoardDateQuestion { get; set; }
+      public string PreviousHeadTeacherBoardLink { get; set; }
+      public string TrustReferenceNumber { get; set; }
+      public string NameOfTrust { get; set; }
+      public string SponsorReferenceNumber { get; set; }
+      public string SponsorName { get; set; }
+      public string AcademyTypeAndRoute { get; set; }
+      public DateTime? ProposedAcademyOpeningDate { get; set; }
+      public bool? SchoolAndTrustInformationSectionComplete { get; set; }
+
+      [Column(TypeName = "decimal(38, 2)")] public decimal ConversionSupportGrantAmount { get; set; }
+
+      public string ConversionSupportGrantChangeReason { get; set; }
+
+      //general info
+      public string SchoolPhase { get; set; }
+      public string AgeRange { get; set; }
+      public string SchoolType { get; set; }
+      public int? ActualPupilNumbers { get; set; }
+      public int? Capacity { get; set; }
+      public string PublishedAdmissionNumber { get; set; }
+      [Column(TypeName = "decimal(38, 3)")] public decimal? PercentageFreeSchoolMeals { get; set; }
+      public string PartOfPfiScheme { get; set; }
+      public string ViabilityIssues { get; set; }
+      public string FinancialDeficit { get; set; }
+      public string DiocesanTrust { get; set; }
+
+      [Column(TypeName = "decimal(38, 3)")]
+      public decimal? PercentageOfGoodOrOutstandingSchoolsInTheDiocesanTrust { get; set; }
+
+      [Column(TypeName = "decimal(38, 3)")] public decimal? DistanceFromSchoolToTrustHeadquarters { get; set; }
+      public string DistanceFromSchoolToTrustHeadquartersAdditionalInformation { get; set; }
+      public string MemberOfParliamentParty { get; set; }
+      public string MemberOfParliamentName { get; set; }
+
+      public bool? GeneralInformationSectionComplete { get; set; }
+
+      //school performance ofsted information
+      public string SchoolPerformanceAdditionalInformation { get; set; }
+
+      // rationale
+      public string RationaleForProject { get; set; }
+      public string RationaleForTrust { get; set; }
+      public bool? RationaleSectionComplete { get; set; }
+
+      // risk and issues
+      public string RisksAndIssues { get; set; }
+      public string EqualitiesImpactAssessmentConsidered { get; set; }
+      public bool? RisksAndIssuesSectionComplete { get; set; }
+
+      // legal requirements
+      public string GoverningBodyResolution { get; set; }
```

Comment:
Small thing: can we stick to the spacing that's used here already.
In other words no carriage return between each field.

Reply:
I don't understand what you mean - seems the same as all of the rest of the file... ?